tests/requirements.txt

```diff
@@ -1,5 +1,5 @@
 coverage>=3.7.1
 nose>=1.3.7
 responses>=0.4.0
--e git://github.com/CenterForOpenScience/pydocx.git#egg=pydocx
+-e git+https://github.com/CenterForOpenScience/pydocx.git#egg=pydocx
```

Comment:
GitHub [dropped support](https://github.blog/2021-09-01-improving-git-protocol-security-github/) for unencrypted protocols.